plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php

```diff
@@ -71,6 +71,12 @@ protected function doDispatch(KalturaBatchJob $job, KalturaIntegrationJobData &$
 	
 	protected function doClose(KalturaBatchJob $job, KalturaIntegrationJobData &$data, KalturaVoicebaseJobProviderData $providerData)
 	{
+		$entryId = $providerData->entryId;
+		KalturaLog::debug("checking entry id - $entryId");
+		$this->clientHelper = VoicebasePlugin::getClientHelper($providerData->apiKey, $providerData->apiPassword);
+		if($this->clientHelper->checkExitingExternalContent($entryId))

```

Comment:
@shpiNi  I believe that this is a typo - probably meant to be checkExistingExternalContent, please change here and on the VoiceBaseClientHelper class